samples/generatedkey-sample/pom.xml

```diff
@@ -47,6 +47,11 @@
             <artifactId>kafka-encryption-generatedkey</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.quicksign</groupId>
+            <artifactId>kafka-encryption-keyrepository</artifactId>
+        </dependency>
+
```

Comment:
You are in "generatedkey-sample", please don't mix the samples.

Reply:
Yes, it's a mistake, don't worry :)